README.md

```diff
@@ -29,60 +29,34 @@ The current const_fn requires Rust 1.31 or later.
 
 ## Examples
 
-When using like the following functions to control unstable features:
-
-```toml
-[features]
-const_unstable = []
-```
-
-It can be written as follows:
-
 ```rust
-#![cfg_attr(feature = "const_unstable", feature(const_fn))]
 use const_fn::const_fn;
 
-pub struct Foo<T> {
-    x:T,
+// 1.36 and later compiler (including beta and nightly)
+#[const_fn("1.36")]
+pub const fn version() {
+    /* ... */
```

Comment:
It currently only accepts string literals as version requirements, like cfg_version does.

Reply:
Filed #19